### PR TITLE
Insert long email into config set

### PR DESCRIPTION
**Pull request by pwalczysko**
A very tentative attempt to insert a long email text into the config of the demo server.

cc @manics @joshmoore 

see https://openmicroscopy.slack.com/archives/C0K5ME1EW/p1610560470190100?thread_ts=1610549673.183000&cid=C0K5ME1EW

### CODE REVIEW

File: templates/omero-web-config-signup.j2 [hunk 1]
```diff
@@ -10,4 +10,20 @@ config set -- omero.web.signup.group.name %Y-%m
 config set -- omero.web.signup.group.templatetime true
 config set -- omero.web.signup.email.enabled true
 config set -- omero.web.signup.helpmessage 'If you have trouble creating an account please contact {{ omero_server_mail_from | default('help@example.org') }}'
-config set -- omero.web.signup.email.body 'Your login details for {{ inventory_hostname }} are:\n\nusername: {username}\npassword: {password}\n'
+config set -- >-
```
Comment (manics) [inline templates/omero-web-config-signup.j2:13]: Sorry, I made a mistake, this is a Jinja2 file so YAML syntax doesn't work. What would work is putting the variable in the playbook yml file, and using a variable here. E.g.
```suggestion
config set -- omero.web.signup.email.body '{{ omero_signup_email_body }}'
```
and in the playbook vars set
```yml
omero_signup_email_body: >-
  Thank you for creating an account on demo.openmicroscopy.org server.
  Your login details are \n\n
  username: {username}\n
  ...
```

File: ome-demoserver.yml [hunk 2]
```diff
@@ -384,6 +384,24 @@
     omero_cli_render_release: "{{ omero_cli_render_release_override | default('0.6.1') }}"
     omero_metadata_release: "{{ omero_cli_metadata_release_override | default('0.5.0') }}"
 
+    # Signup
+    omero_signup_email_body: >-
```
Comment (sbesson) [inline ome-demoserver.yml:388]: While trying to deploy to `pub-omero` this failed with

```
Jan 15 13:25:42 pub-omero.openmicroscopy.org systemd[1]: Starting OMERO.web...
Jan 15 13:25:43 pub-omero.openmicroscopy.org omero[22766]: usage: /opt/omero/web/venv3/bin/omero config set [-h] [--report] [-f FILE]
Jan 15 13:25:43 pub-omero.openmicroscopy.org omero[22766]: KEY [VALUE]
Jan 15 13:25:43 pub-omero.openmicroscopy.org omero[22766]: /opt/omero/web/venv3/bin/omero config set: error: unrecognized arguments: you for creating an account on demo.openmicroscopy.org server. Your login detai...opy.orgn and l
Jan 15 13:25:43 pub-omero.openmicroscopy.org systemd[1]: omero-web.service: control process exited, code=exited status=2
Jan 15 13:25:43 pub-omero.openmicroscopy.org systemd[1]: Failed to start OMERO.web.
Jan 15 13:25:43 pub-omero.openmicroscopy.org systemd[1]: Unit omero-web.service entered failed state.
Jan 15 13:25:43 pub-omero.openmicroscopy.org systemd[1]: omero-web.service failed.
```

File: templates/omero-web-config-signup.j2 [hunk 3]
```diff
@@ -10,4 +10,4 @@ config set -- omero.web.signup.group.name %Y-%m
 config set -- omero.web.signup.group.templatetime true
 config set -- omero.web.signup.email.enabled true
 config set -- omero.web.signup.helpmessage 'If you have trouble creating an account please contact {{ omero_server_mail_from | default('help@example.org') }}'
-config set -- omero.web.signup.email.body 'Your login details for {{ inventory_hostname }} are:\n\nusername: {username}\npassword: {password}\n'
+config set -- omero.web.signup.email.body '{{ omero_signup_email_body }}'
```
Comment (sbesson) [inline templates/omero-web-config-signup.j2:13]:
```suggestion
config set -- omero.web.signup.email.body {{ omero_signup_email_body }}
```

The double quoting (in this file and in the vars section) was responsible for https://github.com/ome/prod-playbooks/pull/301/files#r558310403.

**Comment by pwalczysko** [comment]
The trailing whitespace is not making molecule happy. Will push fixes.

**Review by sbesson — APPROVED** [review]
Deployed the latest version of this playbook onto `pub-omero`. The sign-up email reads as expected.

Let's discuss the deployment onto `demo` on Monday.

**Comment by pwalczysko** [comment]
> Deployed the latest version of this playbook onto `pub-omero`. The sign-up email reads as expected.
> 
> Let's discuss the deployment onto `demo` on Monday.

Thank you. Tested the signup email too. Works fine for me.

**Comment by sbesson** [comment]
Deployed on `demo.openmicroscopy.org`

**Pull request merged**